gov_uk_dashboards/components/plotly/graph.py

```diff
@@ -0,0 +1,19 @@
+"""Create a graph"""
+from dash import dcc
+
+
+def graph(element_id: str, figure: any, style: dict = None):
+    """
+    Create a responsive dash graph.
+    Forces the default height of 450px that plotly uses for
+    graphs if style is not specified.
+    """
+    if not style:
+        return dcc.Graph(
+            id=element_id, figure=figure, responsive=True, style={"height": "450px"}
+        )
```

Comment:
Stylistic suggestion:
```suggestion
        style = {}
```
I feel this might make the function a little simpler to understand.

Reply:
Tru dat, tru dat